src/style/cell-styles.css

```diff
@@ -0,0 +1,184 @@
+/*  cell and cell container styles*/
+
+div.cell-container {
+  padding-top: 5px;
+  padding-bottom: 5px;
+  padding-right: 5px;
+  padding-left: 5px;
+  margin-top: 15px;
+  margin-bottom: 15px;
+  outline: #f4f4f4 1px solid;
+  width: calc(100%);
+  /* put the .cell-menu-container and .cell-row-container in a row */
+  display: flex;
+  flex-direction: row; 
+}
+
+/* These may need to be integrated into the presentation mode style sheets
+but I don't see why we would have to do this immediately.
+*/
+
+div#cells.presentation div.cell-container {
+  margin: 0 auto; 
+  outline: none;
+  padding: 0px;
+}
+
+div#cells.presentation .collapse-button {
+  display: none;
+  height: 0px
+}
+
+div#cells.presentation div.cell-row.HIDDEN {
+  display: none;
+}
+
+div#cells.presentation div.cell-menu-container {
+  display: none;
+  height: 0px;
+  width: 0px !important;
+}
+
+.cell-menu-container {
+    width: 50px;
+}
+
+.cell-type-label {
+  text-align: right;
+  color: hsl(275, 12%, 82%);
+  cursor: pointer;
+  padding: 0px 6px 0px 0px;
+}
+.cell-type-label:hover {
+  background: #fbfbfb;
+  outline: 1px solid #ddd;
+  color: hsl(275, 4%, 40%);
+}
+
+div.cell-row-container {
+  width: calc(100% - 50px); /* 100% minus cell-menu-contatiner width*/
+}
+
+div.cell-row {
+  display: flex;
+  padding: 1px;
+}
+
+div#cells.editor div.cell-container.markdown div.cell-row.output {
+  overflow-x:hidden;
+}
+
+div.collapse-button {
+  margin-right: 3px;
+  margin-left: 3px;
+  width: 18px;
+  height: 100%;
+}
+
+div.collapse-button:hover {
+  background: #fbfbfb;
+  outline: 1px solid #ddd;
+  cursor: pointer;
+}
+
+div.HIDDEN div.collapse-button-tooltip-wrapper {
+    width: calc(100% - 3px); /* 100% minus a .collapse-button width margin-right*/
+
+}
+
+div.HIDDEN div.collapse-button {
+    width: calc(100%);
+    /* when collapsed, make the button visible*/
+  background: #fdfdfd;
+  outline: 1px solid #ddd;
+  cursor: pointer;
+}
+
+div.main-component {
```

Comment:
This style selection can also be accompanied with the following presentation mode override in the presentation stylesheet, toward the top:

```css
/* in the editor view, there is 24px less room for the collapse-button. Let's get it back. */
div#cells.presentation div.main-component {
  width: 100%;
}
```

To make it so the `main-component` width is 100%, not `calc(100%  - 24px)`. This will make it easy to center / change the presentation style for users who are mucking around with the width of the main components.